### PR TITLE
add audio-recorder

### PR DESCRIPTION
**Pull request by dimahalaiskiy**

### CODE REVIEW

File: exadel-trainee-app/src/app/models/speaking-test/speaking-test.component.ts [hunk 1]
```diff
@@ -1,10 +1,64 @@
...
+
+  chunks:Blob[] = [];
+  audioFiles:{src: SafeUrl}[] = [];
+
+  constructor(private cd: ChangeDetectorRef, private dom: DomSanitizer) {
+
+  }
+
+  async ngOnInit() {
+    let stream = null;
```
Comment (Inunor) [inline exadel-trainee-app/src/app/models/speaking-test/speaking-test.component.ts:22]: Please remove all these console.logs

**Comment by Inunor** [comment]
Pr name is wrong, it doesn't look like fix, it's a feature

**Comment by IrynaFiadchuk** [comment]
[FrontEnd Tasks](https://trello.com/c/2DqeegD1/14-frontend-tasks)

**Review by RomPavel — APPROVED** [review]

File: exadel-trainee-app/src/app/models/speaking-test/speaking-test.component.ts (hunk 1 above)
Comment (dimahalaiskiy) [reply exadel-trainee-app/src/app/models/speaking-test/speaking-test.component.ts:21]: Done

**Pull request merged**